asllib/doc/Statements.tex

```diff
@@ -1554,33 +1604,17 @@ \subsubsection{Formally}
   \vepattern \eqdef \EPattern(\vezero, \vpattern)\\
   \vcond \eqdef \choice{\vwhere = \langle\vewhere\rangle}{\EBinop(\BAND, \vepattern, \vewhere)}{\vepattern}
 }{
-  \casetocond(\vezero, \vcase, \vtail) \typearrow \overname{\SCond(\vcond, \vstmt, \vtail)}{\news}
+  \casetocond(\vezero, \vcase, \vtail) \astarrow \overname{\SCond(\vcond, \vstmt, \vtail)}{\news}
 }
 \end{mathpar}
 
-\subsubsection{TypingRule.SCase\label{sec:TypingRule.SCase}}
-\subsubsection{Prose}
-All of the following apply:
-\begin{itemize}
-  \item $\vs$ is a case statement;
-  \item applying $\desugarcasestmt$ to $\vs$ transforms $\vs$ to a conditional statement $\vsp$;
-  \item annotating $\vsp$ in $\tenv$ yields $(\news, \newtenv, \vses)$\ProseOrTypeError.
-\end{itemize}
-\subsubsection{Formally}
-\begin{mathpar}
-\inferrule{
-  \astlabel(\vs) = \SCase\\
-  \desugarcasestmt(\vs) \typearrow \vsp\\
-  \annotatestmt(\tenv, \vsp) \typearrow (\news, \newtenv, \vses) \OrTypeError
-}{
-  \annotatestmt(\tenv,\vs) \typearrow (\news, \newtenv, \vses)
-}
-\end{mathpar}
-\CodeSubsection{\SCaseBegin}{\SCaseEnd}{../Typing.ml}
+\subsection{Typing}
+Since case statements are transformed into other statements,
```

Comment:
above the term "desugared-away" is used - here the term "transformed" is used. Please pick one (and define it)